pkgs/games/pegasus-frontend/default.nix

```diff
@@ -0,0 +1,43 @@
+{ lib
+, fetchFromGitHub
+, stdenv
+, cmake
+, libsForQt5
+, wrapQtAppsHook
+, SDL2
+, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "pegasus-frontend";
+  version = "22-05-2023-6421d7a";
+
+  src = fetchFromGitHub {
+    owner = "mmatyas";
+    repo = "pegasus-frontend";
+    rev = "6421d7a75d29a82ea06008e4a08ec14e074430d9";
+    sha256 = "sha256-mwJm+3zMP4alcis7OFQUcH3eXlRTZhoZYtxKrvCQGc8=";
+    fetchSubmodules = true;
+  };
+
+  nativeBuildInputs = [ cmake wrapQtAppsHook ];
+
+  buildInputs = [
+    libsForQt5.qt5.qtbase
+    libsForQt5.qt5.qtmultimedia
+    libsForQt5.qt5.qtsvg
+    libsForQt5.qt5.qttools
+    libsForQt5.qt5.qtgraphicaleffects
+    libsForQt5.qt5.qtx11extras
+    sqlite
+    SDL2
+  ];
+
+  meta = with lib; {
+    description = "A cross platform, customizable graphical frontend for launching emulators and managing your game collection.";
+    homepage = "https://pegasus-frontend.org/";
+    license = licenses.gpl3;
```

Comment:
`gpl3` is unclear, use `gpl3Plus` or `gpl3Only`. See https://discourse.nixos.org/t/lib-licenses-gpl3-co-are-now-deprecated/8206 for more details.